_extras/guide.md

```diff
@@ -137,6 +137,10 @@ This idiom violates the recommendation not to repeat variables names, but since
 
 * The Pan-STARRS join exercise is likely to feel scary to a lot of students - they have only seen one example. Emphasize the building blocks of a join e.g. FROM table1 JOIN table2 ON table1.common_column=table2.common_column. It is also worth pointing out that common_columns will not have the same names in this case
 
+* In case a student asks about the extra `unnamed` column in the CSV section here's the explanation. You may notice that all Pandas `DataFrame`s have an index column which was not part of the original table definition. 
```

Comment:
```suggestion
* In case a student asks about the extra `unnamed` column in the CSV section here's the explanation:
You may notice that all Pandas `DataFrame`s have an index column which was not part of the original table definition. 
```